tests/functests/instancetype_test.go

```diff
@@ -251,15 +251,15 @@ func createInstancetype(cpu int, name, memory string) {
 	Expect(err).ToNot(HaveOccurred())
 }
 
-func randomVM(instancetype *v1.InstancetypeMatcher, preference *v1.PreferenceMatcher, running bool) *v1.VirtualMachine {
+func randomVM(instancetype *v1.InstancetypeMatcher, preference *v1.PreferenceMatcher, runStrategy v1.VirtualMachineRunStrategy) *v1.VirtualMachine {
 	name := "test-vm-" + k8srand.String(5)
 	return &v1.VirtualMachine{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      name,
 			Namespace: testNamespace,
 		},
 		Spec: v1.VirtualMachineSpec{
-			Running:      ptr.To(running),
+			RunStrategy:  ptr.To(runStrategy),
```

Comment:
```suggestion
			RunStrategy:  &runStrategy,
```

This should work too.